test/cli/test_cli_functions.sh

```diff
@@ -284,6 +284,23 @@ function obc_cycle {
     echo_time "✅  obc cycle is done"
 }
 
+function crd_cycle {  
+    local crd 
+    echo_time "Starting the crd cycle"
+    test_noobaa timeout crd create
+    test_noobaa timeout crd wait
+    crd_array=($(kubectl get crd | awk '{print $1}' | grep -v "NAME"))
+    echo_time "${crd_array[*]}"
+    for crd in ${crd_array[@]}
+    do 
+        test_noobaa timeout crd status
```

Comment:
`noobaa crd status` returns all the crds, we don't need it inside the loop

we should check that each card expected is present here, after we do status